src/community/features-templating/features-templating-wfs/src/main/java/org/geoserver/featurestemplating/wfs/HTMLTemplateResponse.java

```diff
@@ -115,4 +119,35 @@ protected void afterEvaluation(TemplateOutputWriter writer, RootBuilder root, Fe
             throws IOException {
         super.afterEvaluation(writer, root, feature);
     }
+
+    @Override
+    protected boolean canHandleInternal(Operation operation) {
```

Comment:
Please add some comments regarding what this method does on conditions and operations.  Even better if we can split some of them on private methods.

Reply:
fixed